src/panda_eup/pbd_interface.py

```diff
@@ -8,15 +8,18 @@
 from panda_pbd.msg import UserSyncGoal, MoveToContactGoal, MoveToEEGoal
 from panda_pbd.srv import MoveFingersRequest, ApplyForceFingersRequest
 from sensor_msgs.msg import JointState
+import numpy as np
+import random
 
 
 class PandaPBDInterface(object):
-    def __init__(self):
+    def __init__(self, robotless_debug):
         self.program = pp.PandaProgram('A Panda Program')
-
+        self.robotless_debug = robotless_debug
         self.last_pose = None
         self.last_gripper_width = None
         self.relaxed = False
+        self.frozen = False
```

Comment:
Looks to me that self.frozen is always not self.relaxed. Can you remove this redundancy?